internal/pkg/describe/app.go

```diff
@@ -53,8 +54,15 @@ func (a *App) HumanString() string {
 	if deploy.LatestAppTemplateVersion != a.Version {
 		availableVersion = color.Yellow.Sprintf("(latest available: %s)", deploy.LatestAppTemplateVersion)
 	}
+	if a.URI == "" {
+		a.URI = "N/A"
+	}
+	if a.PermissionsBoundary == "" {
+		a.PermissionsBoundary = "N/A"
+	}
 	fmt.Fprintf(writer, "  %s\t%s %s\n", "Version", a.Version, availableVersion)
 	fmt.Fprintf(writer, "  %s\t%s\n", "URI", a.URI)
```

Comment:
Unrelated to your PR but I feel like we should rename this from "URI" to "Domain" in a separate PR